pom.xml

```diff
@@ -198,10 +198,20 @@
         <dependency>
             <groupId>org.hamcrest</groupId>
             <artifactId>hamcrest-core</artifactId>
+            <!-- Without Spring Boot this must be explicit -->
+            <version>2.2</version>
         </dependency>
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-core</artifactId>
+            <!-- Without Spring Boot this must be explicit -->
+            <version>4.6.1</version>
```

Comment:
Version for mockito is defined in depencyManagement.

Reply:
Moved it. Nevertheless I don't see any advantage of the split between dependencyManagement and dependencies without child modules. The few and isolated pinned versions for test dependencies seem straightforward. Thus they could easily end up in a single <dependencies> block.